### DISCUSSION TITLE
Adding iOS 17 support

### DISCUSSION THREAD

**Discussion by ghost** [ChefKissInc/Inferno#12] [Q&A] [answered]
Hey. I was wondering what areas need work to add support for iOS 17? @VisualEhrmanntraut

**Comment by VisualEhrmanntraut** [comment]
Not sure, at the very least the kernel loading and RTBuddy logic

**Reply by ghost** [reply]
> Not sure, at the very least the kernel loading and RTBuddy logic

I see.
What iOS version are you using with this repo's version?

**Reply by VisualEhrmanntraut** [reply, marked as answer]
> > Not sure, at the very least the kernel loading and RTBuddy logic
> 
> I see.
> What iOS version are you using with this repo's version?

iOS 14, specifically 18A5351d
